### ISSUE TITLE
corrupted my 3ds sd card lost all my games

### ISSUE THREAD

**Issue by chokymilkyway** [erievs/FourthTube#60]
so this more of a warning for people but this app crashes a lot atleast on a old 3ds and one of those crashes resulted in my sd card being corrupted and losing all my games

**Comment by erievs** [comment]
Yeah seems very unlikely it's caused by fourthtube, I am almost certain your sd card was dying already.

**Comment by erievs** [comment]
It's unfortunate but doubt it's caused by this.
